styles.css

```diff
@@ -406,3 +406,133 @@ hr {
 .card-multi {
   height: 590px;
 }
+
+/* contact form start */
+
+.card-contact {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  background-color: #6070ff;
+  height: 779px;
+  width: 375px;
```

Comment:
- [ ] [OPTIONAL] You did a good job implementing the given design, but I would suggest you avoid using fixed widths and height as much as you can. This is because when you are working on any project, you want it to be as much responsive as possible, so it can fit on any browser, and fixed values like this one are not the best options, personally, I suggest using either `rem` or `em` units so you can make your project look good on any screen size.

Reply:
Ok, will follow the suggestion.